### PR TITLE
[DE] Add `Music theory/Beat`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
As already described in #10176, I used `Strich` to translate `Tick`.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Music_theory/Beat/de.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+# Beat
+
+Ein **Beat** (auch *Grundschlag* genannt) ist ein gleichmäßig wiederkehrender Schlag oder eine Zeiteinheit in der Musik, die den Rhythmus erzeugt, zu dem eine Person beim Zuhören klopfen oder ihren Kopf nicken würde.
+
+![Screenshot der Editor-Zeitleiste im Spiel](img/timeline.png "Jeder weiße Strich repräsentiert einen Beat in der Musik")
+
+Beats haben normalerweise dieselbe Länge wie eine Viertelnote (abhängig vom Nenner der [Taktart](/wiki/Music_theory/Time_signature)), die zwischen zwei aufeinanderfolgenden weißen Linien auf der [Zeitleiste des Beatmap-Editors](/wiki/Client/Beatmap_editor/Timelines) liegen. Die gleiche Anzahl von Notenwerten wird in einem [Takt](/wiki/Music_theory/Measure) gruppiert, der mit einem [Downbeat](/wiki/Music_theory/Downbeat) beginnt (dargestellt von einem großen weißen Strich). Die Zeit zwischen jedem Beat hängt vom [Tempo](/wiki/Music_theory/Tempo) ab.
+
+Andere Noten werden durch das Teilen oder Multiplizieren ihrer Länge dargestellt. Der [Taktteiler](/wiki/Client/Beatmap_editor/Beat_snap_divisor) im Beatmap-Editor macht das automatisch für einen Mapper und ermöglicht die Änderung der zeitlichen Granularität im Handumdrehen.
```
Comment (FreezyLemon) [inline wiki/Music_theory/Beat/de.md:26]: Maybe, to clarify:

```suggestion
Andere Noten werden durch das Teilen oder Multiplizieren der Länge eines Beats dargestellt. Der [Taktteiler](/wiki/Client/Beatmap_editor/Beat_snap_divisor) im Beatmap-Editor macht das automatisch für einen Mapper und ermöglicht die Änderung der zeitlichen Granularität im Handumdrehen.
```

File: wiki/Music_theory/Beat/de.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+tags:
+  - beats
+  - white line
+  - white lines
+  - white tick
+  - white ticks
+  - whole beat
+  - whole beats
+  - ganze Note
+  - ganze Note
```
Comment (FreezyLemon) [inline wiki/Music_theory/Beat/de.md:11]: Did you mean:

```suggestion
  - ganze Noten
```

**Review by FreezyLemon — APPROVED** [review]
LGTM. Only nitpicks

File: wiki/Music_theory/Beat/de.md (hunk 2 above)
Comment (The-Last-Cookie) [reply wiki/Music_theory/Beat/de.md:11]: Good catch, classic copy-paste 👀

**Review by FreezyLemon — APPROVED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**